drake/systems/@DynamicalSystem/DynamicalSystem.m

```diff
@@ -719,5 +719,8 @@
   
   properties (Access=public)
     warning_manager;
-  end    
+  end
+  properties (SetAccess=protected)
+    initial_state = [];
```

Comment:
getInitialState causes all sorts of Simulink-related problems when there are constraints. The sequence is something like this:

1) DynamicalSystem.simulate() is given x_0 and calls some Simulink initialization methods
2) Simulink calls getInitialState, but without x_0
3) getInitialState uses some randomness and then tries to solve nonlinear programs to resolve constraints. If x=0 is in contact, this can regularly fail, wasting a whole lot of time and causing errors.
4) The actual returned result from getInitialState is entirely ignored by Simulink anyway.